widget/embedded/src/components/Slippage/Slippage.styles.ts

```diff
@@ -19,6 +20,12 @@ export const Head = styled('div', {
 });
 
 export const SlippageTooltipContainer = styled('div', {
-  maxWidth: '300px',
+  maxWidth: '280px',
   padding: '$10',
 });
+
```

Comment:
Can you fix this alignment issue? it is also problematic in next. I think The easiest way is to change some padding to the first item.
Please also change 'Setting' to 'Settings' in page title.
 
![Screenshot from 2023-12-11 14-06-32](https://github.com/rango-exchange/rango-client/assets/99180934/fc2d5aeb-3a41-4262-ad3c-dd914718d66e)